indexs/banner.html

```diff
@@ -0,0 +1,40 @@
+<!DOCTYPE html>
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Banner</title>
+    <link rel="stylesheet" href="../estilos/banner.css" />
+  </head>
+  <body>
+    <h1>My Banner</h1>
+    <div class="cards">
+      <div class="cards__1 cards--all">
+        <h3>EVERLANE</h3>
```

Comment:
I see that each "brand name" title uses a different heading tag. For better semantic practices, it's better to use a single heading level (e.g., h2 or h3) for all brand names, and apply classes to differentiate styles or target them via the parent container.